crates/pessimistic-proof/src/proof.rs

```diff
@@ -148,17 +149,20 @@ pub fn generate_pessimistic_proof(
         nullifier_root: prev_nr,
     } = initial_network_state.roots();
     let prev_pessimistic_root = keccak256_combine([prev_lbr, prev_nr]);
+    debug!("prev_pesssimistic_root: {}", Hash(prev_pessimistic_root));
```

Comment:
```suggestion
    debug!("prev_pessimistic_root: {}", Hash(prev_pessimistic_root));
```

do we want to keep these logs?

Reply:
No, those are debug logs for @goran-ethernal

Reply:
Yeah, you can remove them.